Makefile

```diff
@@ -49,6 +49,29 @@ u-bmc:
 	go get
 	go build
 
+# Linux tree comes from the OpenBMC branch, not the official kernel.org
+LINUX_VERSION	:= 5.2.14
+LINUX_DIR	:= linux-$(LINUX_VERSION)
+LINUX_TAR	:= linux-$(LINUX_VERSION).tar.gz
+LINUX_URL	:= https://github.com/openbmc/linux/archive/v$(LINUX_VERSION).tar.gz
```

Comment:
I would assume the patches are applied by rebasing the patches upon branches. If that's the case the tags will always be prestine upstream kernel trees without any OpenBMC patches.

I spot-checked one tag and it appears to be the case. If you find counter-evidence I'm happy to consider this approach, but I think we will need an alternative.

Reply:
I've also confirmed that the openbmc "release" 5.2.14 kernel is identical to the kernel.org 5.2.14. Wonder why they have their own tree of them...

So we need to see if there is an actual release tar file somewhere.